spec/system/user_preferences_navigation_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+describe 'Redesigned user page navigation menu', type: :system, js: true do
+  fab!(:user) { Fabricate(:user) }
+  let(:everyone_group) { Group[:everyone] }
+
+  describe "when visiting the user's preferences page with redesigned user page nav enabled" do
+    it 'should allow the user to scroll the horizontal navigation menu when window width is narrow' do
+      everyone_group.add(user)
```

Comment:
Ran into this same issue when doing the `personal_message_enabled_groups` stuff. Kind of annoying, wish we could do it in the fabricator but then that affects other tests that expect the user to be in no group by default. Another way to do it is by calling `Group.refresh_automatic_groups!`. No need to change here just FYI